00-Login/README.md

```diff
@@ -28,32 +27,18 @@ You should see the `A0Auth0` pod being installed and linked to the sample app.
 
 1. Open the [Auth0 dashboard](https://manage.auth0.com/dashboard/) and click **Applications.**
 2. Select your existing Application from the list or click **Create Application** at the top to create a new one of type **Native**. 
-3. On the **Settings** tab for the Application, copy the "Client ID" and "Domain" values and paste them in the `app/auth0-credentials.js` file. This file is being used in the `app/index.js` file to instantiate the library. You can also change and do that manually this way:
-
-```js
-import Auth0 from 'react-native-auth0';
+3. On the **Settings** tab for the Application, copy the "Client ID" and "Domain" values and paste them in the `app/auth0-credentials.js` file. This file is being used in the `app/index.js` file to instantiate the library.
+4. In the **Allowed Callback URLs** field, paste in the text below and replace `YOUR_DOMAIN` with the Domain from step 3. These URLs are required for the authentication result to be redirected from the browser to the app:
 
-const config = {
-  domain: 'YOUR_DOMAIN',
-  clientId: 'YOUR_CLIENT_ID'
-}
-
-const auth0 = new Auth0(config);
+```
+com.auth0samples://YOUR_DOMAIN/ios/com.auth0samples/callback, 
+com.auth0samples://YOUR_DOMAIN/android/com.auth0samples/callback
 ```
 
-### Deep linking setup
```

Comment:
This whole section, minus the part about Android, was informational and not necessary for running the sample. Removed for simplicity and clarity.